### PR TITLE
Update samples to use preview 2 packages

### LINKED ISSUE #164: Update samples to use Iot Package
This project should be updated ...

https://github.com/dotnet/iot/blob/master/samples/led-more-blinking-lights/led-more-blinking-lights.csproj#L14

The README should be updated, too: https://github.com/dotnet/iot/blob/master/src/devices/README.md#binding-distribution

### PR DESCRIPTION
**Pull request by joperezr**
Fixes #164 

cc: @richlander

### CODE REVIEW

File: samples/led-blink/nuget.config [hunk 1]
```diff
@@ -2,7 +2,6 @@
 <configuration>
```
Comment (richlander) [inline samples/led-blink/nuget.config]: probably just delete this whole file??

File: samples/led-more-blinking-lights/led-more-blinking-lights.csproj [hunk 2]
```diff
@@ -7,11 +7,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Device.Gpio" Version="0.1.0-prerelease*" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <ProjectReference Include="..\..\src\devices\Mcp3008\Mcp3008.csproj" />
+    <PackageReference Include="System.Device.Gpio" Version="0.1.0-prerelease.19078.2" />
+    <PackageReference Include="Iot.Device.Bindings" Version="0.1.0-prerelease.19078.2" />
```
Comment (richlander) [inline samples/led-more-blinking-lights/led-more-blinking-lights.csproj]: Is it necessary / best practice to reference both packages? I would think that I.D.B would be enough and would ensure that the S.D.G version matched.

Comment (joperezr) [reply samples/led-more-blinking-lights/led-more-blinking-lights.csproj]: yup, you are right, I'll remove the S.D.G one

File: src/devices/README.md [hunk 3]
```diff
@@ -7,16 +7,18 @@ Our vision: the majority of .NET bindings are written completely in .NET languag
...
 * [Bmp280](Bmp280/README.md)
+* [Dhtxx](Dhtxx/README.md)
+* [Hcsr04](Hcsr04/README.md)
 * [Max7219](Max7219/README.md)
-* [Mcp3008](Mcp3008/README.md)
 * [Mcp23xxx](Mcp23xxx/README.md)
+* [Mcp3008](Mcp3008/README.md)
 * [Si7021](Si7021/README.md)
 
 ## Binding Distribution
```
Comment (richlander) [inline src/devices/README.md]: This whole section should now be written, right? At this point, I think we are saying that binding distribution is primarily via nuget and that we will keep the package up to date. Right?

File: src/devices/README.md [hunk 4]
```diff
@@ -7,16 +7,16 @@ Our vision: the majority of .NET bindings are written completely in .NET languag
...
 
-We are currently encouraging source distribution of device bindings. The [Mcp3008 sample](Mcp3008/samples/README.md) references the [Mcp3008 library](Mcp3008/Mcp3008.csproj) within this repo. You can clone this repo and use the sample project reference model to a device binding or copy bindings directly into your project.
-
-We may publish NuGet packages of device bindings at a later date.
+Binding distribution is primarily via the [Iot.Device.Bindings NuGet package](https://www.nuget.org/packages/Iot.Device.Bindings/0.1.0-prerelease.19078.2) that ships via NuGet (for preview and RTM releases) and MyGet (for daily builds with latest bindings). The MyGet package should be updated with every official build, which means that it will take at most 1 day for a binding to be includded in the package once it has been merged to the master branch.
```
Comment (richlander) [inline src/devices/README.md]: Link to the MyGet feed?

**Pull request merged**